### ISSUE TITLE
Merge after CI is green doesn't seem to be working

### ISSUE THREAD

**Issue by vjeux** [sindresorhus/refined-github#1067]
I just installed the Chrome extension and I see a bunch of things that have changed in the UI but when I click on the Squash & Merge button, I don't see the "Wait for successful checks" like in the gif here: https://user-images.githubusercontent.com/1402241/35192861-3f4a1bf6-fecc-11e7-8b9f-35ee019c6cdf.gif

![image](https://user-images.githubusercontent.com/197597/35838917-91b22816-0aa2-11e8-9362-9ac63e641839.png)

Am I doing something wrong? Or is it not enabled for Squash & Merge?

Thanks! I've wanted that feature for so long, I'm so glad that you made it happen :)

**Comment by vjeux** [comment]
Looks like it's working for another repo. The only difference that I can see that the one where it's working is public and the one where it isn't is private. Could that be it?

**Comment by bfred-it** [comment]
The checkbox only appears if there's a yellow CI icon next to the commit, like what it's being circled with the cursor in that GIF.

**Comment by vjeux** [comment]
It's now working on an another PR in that repo. It may not have initialized correctly right after installing or something. I can't repro it not working so I'm going to close it.

**Comment by vjeux** [comment]
I clicked the green button next to the checkbox and it actually merged it and didn't wait for CI tests to pass. Am I supposed to just wait on that screen and if the checkbox is selected it'll automatically merge? Or did I do the right thing and it was supposed to wait?

**Comment by bfred-it** [comment]
Any errors in the console?

**Comment by vjeux** [comment]
I'm sorry I closed the page :(

**Comment by hkdobrev** [comment]
>  Am I supposed to just wait on that screen and if the checkbox is selected it'll automatically merge?

Yes, it would work if you leave the tab open.

**Comment by jonchui** [comment]
I'm having the same issue as @vjeux , but my yellow icon DOES show up beside the last commit:

![image](https://user-images.githubusercontent.com/204868/84327126-d731b280-ab3b-11ea-8988-be3eb450798f.png)

It used to work for me, but i think after switching our repo to `squash and rebase` it stopped working. 

Does anyone know how to debug this ?

**Comment by fregante** [comment]
I've experienced this before. What may be happening (one of them): 

- RGH creates the checkbox, then GitHub regenerates the form, deleting it. We should detect this.
- RGH doesn't detect the yellow dot if it appears a few seconds _after_ the commit.

But I don't know for sure.

**Comment by fregante** [comment]
https://github.com/sindresorhus/refined-github/issues/3084 could be a good solution to safely keep track of the yellow dot (scrapping most of our `pr-ci-status.ts` file)

**Comment by fregante** [comment]
I think at the moment the feature is completely broken since it never appears for me
